manifests/params.pp

```diff
@@ -31,6 +31,8 @@
     $java_source_file         = hiera('wso2::java::source_file')
     $java_user                = hiera('wso2::java::user')
     $java_group               = hiera('wso2::java::group')
+    $service_refresh_file_list = hiera('wso2::service_refresh_file_list', undef)
+    $remove_file_list         = hiera('wso2::remove_file_list', undef)
```

Comment:
It might be better to align the assignment operator to the same horizontal line to be aligned with the previous standard.